rebar.config

```diff
@@ -167,8 +167,13 @@
                     {deps, [{meck, "0.8.12"},
                             {websocket_client, ".*", {git,
                             "git://github.com/aeternity/websocket_client",
-                            {ref, "a4fb3db"}}}
-                           ]}
+                            {ref, "a4fb3db"}}},
+                            {ephemeral, "2.0.4"}
+                           ]},
+                    {pre_hooks, [{"(linux|darwin|solaris|freebsd|netbsd|openbsd)",
+                                  ct, "scripts/standalone_compiler.sh"},
+                                 {"win32", ct, "sh ./scripts/standalone_compiler.sh"}
```

Comment:
If you want to have less trouble in the future for this kind of hook, you could move the shell script execution into a Makefile target, and just call `make mytarget` here. That should work on all platforms.